src/CSET/operators/_utils.py

```diff
@@ -51,7 +51,7 @@ def get_cube_yxcoordname(cube: iris.cube.Cube) -> tuple[str, str]:
     Y_COORD_NAMES = ["latitude", "grid_latitude", "projection_y_coordinate", "y"]
 
     # Get a list of coordinate names for the cube
-    coord_names = [coord.name() for coord in cube.coords()]
```

Comment:
This change makes sense, as the X and Y coordinates should always be dimension coordinates (Except for sonde data?), but out of curiosity was there something that needed this changing?

Reply:
Yes, its because when creating the test loading a cube with time, pressure latitude and longitude coordinates and subsampling like this
` cube = iris.load_cube("tests/test_data/transect_test_umpl.nc")
  cube = cube[:, :, 0, 0]  # Remove spatial dimcoords`
Did not remove the coordinate 'latitude' from the cube (i.e. it becomes a scalar coordinate). We really want the get_cube_yxcoordname to only return values on spatial data, where there are two spatial DimCoords.

Reply:
I'm a little uncertain about why a scalar coordinate isn't a valid latitude/longitude. In principle it should continue to work fine, and represents some point data.

Reply:
I think I'll add a small, specific function which checks if a cube has Dimension coordinates that are spatial than returns True or False, rather than using this existing function to determine if a cube is spatial or not - will create separate issue documenting this.

Reply:
Resolving as I have created a separate function for this.